recipe-chicken-with-apricot.html

```diff
@@ -14,7 +14,7 @@
         <div class="image-section">
             <h3>Dinner: Chicken With Apricot</h3>
             <a href="./images/recipe-chicken-with-apricot.jpg">Click For Larger Image</a>
-            <img src="./images/recipe-chicken-with-apricot.jpg" alt="Recipe: Chicken with Apricots">
+            <img src="./images/recipe-chicken-with-apricot.jpg">
```

Comment:
Please re-add the alt attribute to the image.

Reply:
done, I've re-added it